corehq/apps/app_manager/tests/test_profile.py

```diff
@@ -13,7 +14,9 @@ class ProfileTest(SimpleTestCase, TestFileMixin):
     def setUp(self):
         self.app = Application(build_spec=BuildSpec(
             version='2.7.0'
-        ))
+            ),
+            name=u"TÉST ÁPP"
+        )
 
     def _test_profile(self, app):
         profile_xml = ET.fromstring(app.create_profile())
```

Comment:
Looks like this might be a reasonable place to verify:
```python
profile = app.create_profile()
assert isinstance(profile, bytes), type(profile)
assert u"TÉST ÁPP" in profile.decode('utf-8') # decode is more important than __contains__ here
profile_xml = ET.fromstring(profile)
```